### PR TITLE
Add initialPicker to ColorPicker

### PR DESCRIPTION
**Pull request by nohli**
https://github.com/fluttercandies/flutter_hsvcolor_picker/pull/2 wasn't updated, so I made a new one.

This time with enum `Picker` and not index.

### CODE REVIEW

File: lib/src/color_picker.dart [hunk 1]
```diff
@@ -10,17 +10,34 @@ import 'rgb_picker.dart';
...
     Key? key,
   }) : super(key: key);
 
   final ValueChanged<Color> onChanged;
   final Color color;
 
+  /// The first picker widget that is shown.
+  ///
+  /// Use [Picker.swatches], [Picker.rgb], [Picker.hsv], [Picker.wheel],
+  /// [Picker.paletteHue], [Picker.paletteSaturation] or [Picker.paletteValue].
```
Comment (AlexV525) [inline lib/src/color_picker.dart:38]: Could we remove this or make it like:
```suggestion
  /// See also:
  ///
  ///  * [Picker] Enumeration of pickers.
```

**Review by AlexV525 — COMMENTED** [review]
I think this should be breaking change. Can we have any chances to make it not breaking?

**Comment by nohli** [comment]
> I think this should be breaking change. Can we have any chances to make it not breaking?

@AlexV525 what do you mean by braking change?
The behavior is the same. Only with this PR, users can optionally choose a different initial picker.

**Review by AlexV525 — APPROVED** [review]
LGTM. Turns out I missed something.

**Comment by nohli** [comment]
> LGTM. Turns out I missed something.

Thanks for all your help!

Will try to update pub.dev with the three latest fixes.
Is version 1.0.3 ok?

**Comment by AlexV525** [comment]
Maybe 1.1.0? Also I think the README can be updated too. 👍

**Comment by nohli** [comment]
Ok.

Anything specific you want to change in the readme?

**Comment by AlexV525** [comment]
> Anything specific you want to change in the readme?

For me the only thing on demand is to update the outdated `pub.dartlang.org` link.

**Comment by nohli** [comment]
I removes the link, because:

- it is now in the About section on GitHub.
- the README is primarily for pub.dev, so linking the same url the user is visiting doesn't help too much.

**Pull request merged**